crates/revm/src/inspector.rs

```diff
@@ -197,4 +99,27 @@ impl<DB: Database> Inspector<DB> for NoOpInspector {
     }
 
     fn selfdestruct(&mut self) {}
+
+    /// If needed you can override some of the spec configurations when running with inspector
+    fn override_spec(&self) -> &OverrideSpec {
+        &OVERRIDE_SPEC_DEFAULT
+    }
```

Comment:
Small note: I will probably remove `override_spec` from inspector and just include flag `skip_eip170_contract_code_size_limit` in `env.cfg`. When i first added this i was obsess with making it as fast as possible with macroes and stuff, litlle bit bias decision for this particular case. :)

Reply:
That will work too :)